cmd/Attribution.txt

```diff
@@ -0,0 +1,28 @@
+The following open source projects are referenced by Modbus Device Service Go:
+
+goburrow/modbus (unspecified) - https://github.com/goburrow/modbus
+https://github.com/goburrow/modbus/blob/master/LICENSE
+
+tbrandon/mbserver (MIT) - https://github.com/tbrandon/mbserver
+https://github.com/tbrandon/mbserver/blob/master/LICENSE
```

Comment:
This library also imports https://github.com/goburrow/serial, so we need to include the license(s) for that project here as well.

Reply:
I just checked and that serial library doesn't include any other 3rd party deps in it, so you can just add the line for that project.